x/distribution/abci_app.go

```diff
@@ -10,20 +10,7 @@ import (
 // set the proposer for determining distribution during endblock
 func BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock, k keeper.Keeper) {
 
-	if ctx.BlockHeight() > 1 {
-		previousPercentPrecommitVotes := getPreviousPercentPrecommitVotes(req)
-		previousProposer := k.GetPreviousProposerConsAddr(ctx)
-		k.AllocateTokens(ctx, previousPercentPrecommitVotes, previousProposer)
-	}
-
-	consAddr := sdk.ConsAddress(req.Header.ProposerAddress)
-	k.SetPreviousProposerConsAddr(ctx, consAddr)
-}
-
-// percent precommit votes for the previous block
-func getPreviousPercentPrecommitVotes(req abci.RequestBeginBlock) sdk.Dec {
-
-	// determine the total number of signed power
+	// determine the total power signing the block
 	totalPower, sumPrecommitPower := int64(0), int64(0)
```

Comment:
Can also do `var totalPower, sumPrecommitPower int64` if you don't wanna typecast.